### ISSUE TITLE
The grabber in RoE

### ISSUE THREAD

**Issue by SR-dude** [dhewm/dhewm3#610]
Very early in the dlc RoE you will get the grabber gun. This works like the gravity gun in HL2. You can select it by scrolling the middle mouse button.  But, when you die and reload a save,  you lose it.  You can't select it with the number keys, either. I'm using the mouse/keyboard. 

Like this, you can't finish the game.

**Comment by SR-dude** [comment]
The quick fix is the have the grabber in your hand while your saving your game.

**Comment by DanielGibson** [comment]
What operating system do you use and which version of dhewm3?

**Comment by SR-dude** [comment]
dhewm3 1.5.4.1305 linux-x86_64 Aug  8 2024 21:41:28 using SDL v2.30.6

**Comment by DanielGibson** [comment]
I can't reproduce this.
I just started a new RoE game with the official 1.5.4 Linux binaries and played until the place where you get the grabber.
Then I played just to the next section (killing a few of those flaming heads with the grabber), switched to the pistol and did a quicksave.
Then I jumped into chasm in the next room to kill myself.
![d3_chasm](https://github.com/user-attachments/assets/b470fb5d-8111-452c-b279-f77d2f1b72e6)
Then I loaded the latest quicksave and used the mousewheel to select the grabber, which worked as expected.
